src/components/Button/Button.tsx

```diff
@@ -1,3 +1,78 @@
-import { Button } from "theme-ui"
+/** @jsxRuntime classic /
+/** @jsx jsx */
+import { Button as ThemeUIButton, jsx, SxStyleProp } from "theme-ui"
+import { MouseEvent, ButtonHTMLAttributes, forwardRef } from "react"
+import { shade } from "@theme-ui/color"
+
+export interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  children?: string
+
+  variant?: string
+
+  color?: string
+
+  disabled?: boolean
+
+  onClick?: (event: MouseEvent<HTMLButtonElement>) => void
+}
+
+const Button = forwardRef<HTMLButtonElement, ButtonProps>(function Button(
+  {
+    children,
+    onClick,
+    variant,
```

Comment:
I would add a default value to variant just to be explicit on what is the default variant of the button, as it's an optional prop. 
That makes it easier to understand the component when reading the code and the documentation will show the default value in the prop tables
The same for disabled maybe, mostly for it to show in the documentation, as it's more obvious what the default behavior is

What do you think?